README-ja.md

```diff
@@ -0,0 +1,234 @@
+[![CircleCI](https://img.shields.io/circleci/build/github/xpring-eng/Xpring-JS?style=flat-square)](https://circleci.com/gh/xpring-eng/xpring-js/tree/master)
+[![CodeCov](https://img.shields.io/codecov/c/github/xpring-eng/xpring-js?style=flat-square)]((https://codecov.io/gh/xpring-eng/xpring-js))
+[![Dependabot Status](https://img.shields.io/static/v1?label=Dependabot&message=enabled&color=success&style=flat-square&logo=dependabot)](https://dependabot.com)
+
+# Xpring-JS
+
+Xpring-JSは、Xpring SDKのJavaScriptクライアント側ライブラリです。
+
+## 機能
+Xpring-JSは次の機能を提供します。
```

Comment:
Hi, @U-top-Yoshida , maybe use a colon here? 

> Xpring-JSは次の機能を提供します:

Reply:
@gbarr01 Thank you for your review. yes, a colon will be there.